xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-ui/src/main/java/org/xwiki/test/ui/po/CommentForm.java

```diff
@@ -58,9 +62,32 @@ public WebElement getContentField()
         return getContainer().findElement(By.tagName("textarea"));
     }
 
+    /**
+     * Find the content field and runs an action using the field.
+     *
+     * @param action The action.
+     */
+    public void runOnContentField(Consumer<WebElement> action)
```

Comment:
That's not really a business API for a PO IMO: would be better to have a more explicit API such as `setContent` or something more aligned with what the users would want to perform as action with that field.